botx/models/attachments.py

```diff
@@ -312,105 +312,105 @@ def decode_rfc2397(encoded_content: str) -> bytes:
 EXTENSIONS_TO_MIMETYPES = MappingProxyType(
     {
         # application
-        ".7z": "application/x-7z-compressed",
-        ".abw": "application/x-abiword",
-        ".ai": "application/postscript",
-        ".arc": "application/x-freearc",
-        ".azw": "application/vnd.amazon.ebook",
-        ".bin": "application/octet-stream",
-        ".bz": "application/x-bzip",
-        ".bz2": "application/x-bzip2",
-        ".cda": "application/x-cdf",
-        ".csh": "application/x-csh",
-        ".doc": "application/msword",
-        ".docx": (
+        "7z": "application/x-7z-compressed",
```

Comment:
Pls move this fix to separate commit.

Reply:
This fix to a separate commit. The commit is "fix: files extensions"